examples/extensions/update_sitelink_campaign_extension_setting.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Updates the sitelink campaign extension setting.
+
+Replaces the extension feed items of the sitelink campaign extension setting
+with the given feed item IDs. Note that this doesn't completely remove your
+old extension feed items. See https://developers.google.com/google-ads/api/docs/extensions/extension-settings/overview
+for details.
+"""
+
+
+import argparse
+import sys
+
+from google.api_core import protobuf_helpers
+import google.api_core.path_template
```

Comment:
Just do `from google.api_core import path_template, protobuf_helpers`

Reply:
Actually I realized `path_template` isn't used in the file, can it just be removed?

Reply:
Good catch. Actually, I wanted to use path_template but it didn't seem to be built in a way that worked for the campaign extension setting path. Are those auto-generated?

Reply:
Ah, well you could use it for the campaign extension setting path, but you would need to include the entire structure of the resource name since the helper is generic, so like:

```py
path_template("customers/*/campaignExtensionSettings/*~*", customer_id, campaign_id, extension_type)
```

The helper you're currently using that's part of `campaign_extension_setting_service` is auto-generated and has the static portions already built in, so it's definitely the preferred way to build resource names.